web/src/InstallationProgress.jsx

```diff
@@ -51,31 +42,25 @@ function InstallationProgress() {
   }, []);
 
   const showSubsteps = !!progress.substeps && progress.substeps >= 0;
-  const percentage =
-    progress.steps === 0
-      ? 0
-      : Math.round((progress.step / progress.steps) * 100);
+  const percentage = progress.steps === 0 ? 0 : Math.round((progress.step / progress.steps) * 100);
 
   // FIXME: this is an example. Update or drop it.
   const Messages = () => {
     return (
       <Alert variant="info" isInline isPlain title="Did you know?">
-        You can <a href='#'>read the release notes</a> while the system is being installed.
+        You can <a href="#">read the release notes</a> while the system is being installed.
       </Alert>
     );
-  }
+  };
 
   // FIXME: this is an example. Update or drop it.
   const Actions = () => {
     return (
-      <Button
-        isDisabled
-        onClick={() => console.log("User want to see the summary!") }
-      >
+      <Button isDisabled onClick={() => console.log("User want to see the summary!")}>
```

Comment:
BTW, this is a leftover… but it can be dropped when removing all _layout examples_.